server/controllers/user.js

```diff
@@ -0,0 +1,54 @@
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import uuid from 'uuid/v1';
+import { User } from '../db/models';
+
+dotenv.config();
+class UserController {
+  static async signup(req, res) {
+    const { fullName, email, password } = req.body;
+    try {
+      const dbEmail = await User.findOne({
+        where: {
+          email: email.trim(),
+        },
+      });
+      if (dbEmail) {
+        return res.status(409).json({
+          status: 409,
+          message: 'email already exit',
+        });
+      }
+      const newUser = await User.create({
+        memberID: uuid(),
+        fullName: fullName.trim(),
+        email: email.trim(),
+        password: await bcrypt.hash(password, 10),
+        isAdmin: false,
+      });
+      const token = jwt.sign({
+        email: newUser.email,
+        isAdmin: newUser.isAdmin,
+
+      }, process.env.JWT_KEY);
```

Comment:
Wouldn't it be better if we use helpers for these functions and just call them in the controllers?  That would increase our maintainability I guess.

Reply:
Yeah it would help us safe time